sycl/test/lit.cfg.py

```diff
@@ -67,6 +67,16 @@
 if 'OCL_ICD_FILENAMES' in os.environ:
     config.environment['OCL_ICD_FILENAMES'] = os.environ['OCL_ICD_FILENAMES']
 
+config.available_features.add('opencl-interop')
+if 'SYCL_BE' in os.environ:
+    config.environment['SYCL_BE'] = os.environ['SYCL_BE']
```

Comment:
I don't this code is aligned with the recently committed CUDA back-end.
This patch assumes that `SYCL_BE` is set in the invironment, where as CUDA back-end patch sets this inside this script.
@Ruyk, @Alexander-Johnston, could you take a look, please?
 I think we should align on the process for running LIT tests on machines supporting multiple back-ends.

Reply:
@bjoernknafla